src/liballoc_system/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Whoops, not sure how that got changed, especially since I didn't do any of this work in 2016. (Maybe I slipped and hit Control-A in vim.)  Reverted.